{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/training/profiles/databricks-dev.yaml

```diff
@@ -1,12 +1,12 @@
 # This profile contains config overrides for model development on Databricks
 experiment:
-  name: "/{{cookiecutter.experiment_base_name}}-dev"
+  name: "/dev-{{cookiecutter.experiment_base_name}}"
```

Comment:
I think the reason it doesn't have the parent dir is because in dev workspace, the parent dir doesn't exist. This is the same for running the notebook in test model.

Reply:
Yeah I'm just moving the suffix to prefix to match naming conventions for `staging` and `prod`